packages/entities/entities-plugins/src/definitions/metadata.ts

```diff
@@ -767,6 +767,6 @@ export const CREDENTIAL_SCHEMAS: Record<string, any> = {
  * @returns URL for the plugin icon
  */
 export const getPluginIconURL = (name: string) => {
-  const imageName = PLUGIN_METADATA[name]?.imageName || name
+  const imageName = PLUGIN_METADATA[name]?.imageName || name || 'missing' // default icon is 'missing'
```

Comment:
Sent you this in chat, but I think you could also do something similar to this:

```typescript
export const getPluginIconURL = (name: string) => {
  const imageName = PLUGIN_METADATA[name]?.imageName || name
  const imageHref = new URL(`../assets/images/plugin-icons/${imageName}.png`, import.meta.url).href

  if (!!imageHref && imageHref.includes('undefined')) {
    return new URL('../assets/images/plugin-icons/missing.png', import.meta.url).href
  }

  return imageHref
}
```

Reply:
Implemented in this commit:
https://github.com/Kong/public-ui-components/pull/1314/commits/d306d2dc7c2f81761198e668ab7e1197e19eb92e#diff-eca5dd8c68b49c246b1f9431a95fd878e6e3c2a128cd15b5fdbad4b13f404f5c

Reply:
Ommited `!imageHref` check, since `new URL()` will always generate some kind of URL for us, so doing an empty string check won't be useful.